Lib/idlelib/filelist.py

```diff
@@ -66,7 +70,22 @@ def unregister_maybe_terminate(self, edit):
             del self.dict[key]
         del self.inversedict[edit]
         if not self.inversedict:
-            self.root.quit()
+            try:
+                clip = self.root.clipboard_get().encode('utf-8')
+                if clip and sys.platform[:3] == 'win' and \
+                   len(clip) <= io.DEFAULT_BUFFER_SIZE:
+                    # content size check required to avoid exceeding the pipe bufsize
+                    
+                    # we work directly with stdin as '.communicate' & '.__exit__' call '.wait'
+                    with subprocess.Popen('clip', shell=True, stdin=subprocess.PIPE).stdin as p:
+                        p.write(clip)
+            except (OSError,
+                    _tkinter.TclError):
+                # 'OSError's can probably be ignored (others may not)
+                # 'TclError's are due to an empty clipboard
```

Comment:
```suggestion
                # 'TclError's are due to an empty clipboard.
```
Delete line, make other a proper sentence, as recommended in PEP 8.

Reply:
Done.